tests/integration/utils2.go

```diff
@@ -202,7 +155,7 @@ func init() {
 //
 //	Usage: AssertPanicAndSkipChangeDetection(t, func() { executeTestCase(t, test) })
 func AssertPanicAndSkipChangeDetection(t *testing.T, f assert.PanicTestFunc) bool {
-	if IsDetectingDbChanges() {
+	if change_detector.Enabled {
```

Comment:
nitpick: Perhaps a non-snake case package name or import alias name.

Reply:
Updated to `changeDetector` with an alias.